modules/RTC/TraceablePeerConnection.js

```diff
@@ -63,10 +83,71 @@ function TraceablePeerConnection(
     this.id = id;
 
     /**
+     * Indicates whether or not this instance is used in a peer to peer
+     * connection.
+     * @type {boolean}
+     */
+    this.isP2P = isP2P;
+
+    // FIXME: We should support multiple streams per jid.
+    /**
+     * The map holds remote tracks associated with this peer connection.
```

Comment:
can you document what's being mapped here? like "maps jid to a map of its tracks, keyed by media type" or something